cmd/gotk/get_helmrelease.go

```diff
@@ -51,37 +54,43 @@ func getHelmReleaseCmdRun(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
+	table := tablewriter.NewWriter(os.Stdout)
+	table.SetHeader([]string{"Name", "Revision", "Ready Status", "Message", "Namespace"})
+	table.SetRowLine(true)
```

Comment:
With the below change, this can be moved above the loop that goes through the `list.Items`.